myuw/static/css/mobile.less

```diff
@@ -1399,6 +1416,13 @@ label { font-weight: normal; margin-bottom: auto; }
 				    }
 			    }
 		    }
+		    .card-badge-container .no-top-margin {
```

Comment:
best practice is to use element name/type rather than style description for class names.